---
title: "19163147_MICB425_portfolio"
author: "Jaspreet Gill (19163147)"
date: "version `r format(Sys.time(), '%B %d, %Y')`"
output:
   html_document: 
    toc: yes 
---
# Module 01

## Origins and Earth Systems
### Evidence worksheet 01 
[Whitman *et al* 1998](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)

#### Learning objectives
Describe the numerical abundance of microbial life in relation to ecology and biogeochemistry of Earth systems. 

#### Specific questions
* What were the main questions being asked?  
**How many prokaryotes are present? How much carbon do they produce?**  

* What were the primary methodological approaches used?  
**In aquatic environments, they measured cellular density in cells/mL. In soil, they estimated abundance from unpublished field studies. For subsurface environments, they used two different methods. They either estimated from groundwater data or calculated number of prokaroytes based on the assumptions that average porosity is 3% as well as the average estimated volume of a prokaryotic cell.**  

* Summarize the main results or findings.  
**The total number of prokaryotes is 4-6 x 10^30^. The total cellular carbon on Earth is 350-550 Pg (so prokaryotic carbon is 60-100% of the estimated carbon in plants) and there is 10-fold more nitrogen and phosphorus in prokaryotes than plants. Most prokaryotes are found in open ocea, soil, and oceanic/terrestrial subsurfaces. Prokakryotic cellular production rate is the highest in the open ocean, and their large population size mixed with rapid growth allows for vast genetic diversity.**  

* Do new questions arise from the results?  
**How does the mutation rate affect global cycles? How does turnover rate correlate with mutation rates? How many prokaryotes are in the less-explored areas that were briefly mentioned in the paper?**  

* Were there any specific challenges or advantages in understanding the paper (*e.g.* did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?  
**They presented enough evidence in the paper to supplement the conclusion. The methods could have been a bit more descriptive, especially with the calculations. There could have been more justification in how the calculations were made.**

### Evidence worksheet 02   
[Kasting & Siefert 2002](http://science.sciencemag.org/content/296/5570/1066.long)  

#### Learning objectives  
Comment on the emergence of microbial life and the evolution of Earth systems
    
#### Specific Questions   
* Indicate the key events in the evolution of Earth systems at each approximate moment in the time series. If times need to be adjusted or added to the timeline to fully account for the development of Earth systems, please do so. 

    + 4.6 billion years ago: solar system formed, inner planets had water --> high CO2, high water vapour pressure, high temperatures    
    
    + 4.5 billion years ago: earth hit by an object causing moon to form, earth slightly tilted, night and day cycles created habitats    
    
    + 3.8 billion years ago: meteorites halted - sea water chemistry stabilized (sulfate reduction), early methanogensis  
    
    + 3.5 billion years ago: real evidence of life from carbon isotopes from ocean sedimentary rocks; rubisco - carbon fixation, photosynthesis  
    
    + 3 billion years ago: year-round ice forming at North Pole
    
    + 2.7 billion years ago: possible eukaryotes   
    
    + 2.2 billion years ago: rocks recognized as red beds - oxidation involved    
    + 2.1 billion years ago: appreciable oxygen, switch between mitochondria and chloroplast, increased complexity of life  
    
    + 1.3 billion years ago: first plants appear  
    
    + 550 million years ago: Cambrian explosion, most major animal phyla appeared - rapid expansion of life  
    
    + 550,000 years ago: Archaic humans appeared (*Homo neanderthalensis*)  
    
    + 200,000 years ago: Homo sapiens appear  


* Describe the dominant physical and chemical characteristics of Earth systems at the following waypoints:  

    + Hadean: solar system forming within large cloud of gas around sun, earth initially molten, formation of moons, Heavy Bombardment  
    
    + Archean: no continents, Earth's crust cooled, vast amounts of CO2 in atmosphere, low levels of sunlight so low heat, low oxygen levels  
    
    + Precambrian: major volcanic events altering atmospheric composition; hit by large object causing moon to form  
    
    + Proterozoic: first known glaciations, ending with Snowball Earth (entirely frozen)  
    
    + Phanerozoic: continents drifted apart, eventually connected as Pangaea and then split up again; most biological evolution
    

### Evidence worksheet 03  
[Rockstrom *et al*](https://www.nature.com/articles/461472a)  

#### Learning objectives: 
Evaluate human impacts on the ecology and biogeochemistry of Earth systems.  

#### Specific Questions:
* What were the main questions being asked?  
**How can we redfine preconditions for human development?**  
**What are the consequences of crossing these boundaries?**  

* What were the primary methodological approaches used?  
**Proposed planetary boundaries to meet challenge of maintaining Holocene state which define the safe operating space for humanity**  
**Many thresholds defined by critical value for one or more controlled variables**  
**In general, planetary boundaries are values for control variables that are either at a 'safe' distance from thresholds (for processes with evidence of threshold behaviour) or at dangerous levels (for processes without evidence of thresholds)**  


* Summarize the main results or findings.  
**9 processes:**  
**- climate change**  
**- ocean acidification**  
**- stratospheric ozone depletion**  
**- biogeochemical flow boundaries (interference with nitrogen and phosphorus cycles)**  
**- global freshwater use**  
**- change in land use**  
**- biodiversity loss (terrestrial and marine)**  
**- atmospheric aerosol loading (not yet quantified)**  
**- chemical pollution (not yet quantified)**  
**Three systems (rate of biodiversity loss, climate change, human interference with the nitrogen cycle) have already exceeded their boundaries.**  


* Do new questions arise from the results?  
**How do we define the boundaries for atmospheric aerosol and chemical pollution?**  
**Can the boundaries that have been crossed be reversed?**  
**To what extent are the boundaries linked?**  
**How long does it take to cause significant environmental change?**  

* Were there any specific challenges or	advantages in	understanding the	 paper (e.g.	 did	 the	authors	 provide	 sufficient background information to understand experimental logic, were methods	explained	adequately,	were	any	specific	assumptions	made,	were	conclusions	justified	based	on	the	evidence,	were	the	figures	or tables useful and	easy to understand)?  
**- units of biodiversity loss: why not just number of species/year? why number of species/million species/year?**  
**- explanation for why atmospheric aerosol/chemical pollution is not quantified, or perhaps an estimate would've been helpful**  


### Problem set 01
[Whitman *et al* 1998](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)  

#### Learning objectives  
Describe the numerical abundance of microbial life in relation to the ecology and biogeochemistry of Earth systems.

#### Specific questions  
* What are the primary prokaryotic habitats on Earth and how do they vary with respect to their capacity to support life? Provide a breakdown of total cell abundance for each primary habitat from the tables provided in the text.  
**Aquatic: 1.181 x 10^29^ cells**  
**Soil: 2.556 x 10^29^ cells**    
**Subsurface: 3.8 x 10^30^ cells**   

* What is the estimated prokaryotic cell abundance in the upper 200 m of the ocean and what fraction of this biomass is represented by marine cyanobacterium including Prochlorococcus? What is the significance of this ratio with respect to carbon cycling in the ocean and the atmospheric composition of the Earth?  
**Estimated cell abundance = 360 x 10^26^**  
***Prochlorococcus* = 8% --> 28.8 x 10^26^**  
**Significance for carbon cycling = 8% of upper 200m can photosynthesize --> upper ocean is ideal place for them to be so they can access sunlight**  

* What is the difference between an autotroph, heterotroph, and a lithotroph based on information provided in the text?  
**Autotroph = "self-nourishing"; fix inorganic carbon (eg. CO~2~) into biomass**  
**Heterotroph = consumes and metabolizes inorganic carbon (assimilates inorganic carbon)**  
**Lithotroph = breaks down inorganic substrates**  

* Based on information provided in the text and your knowledge of geography what is the deepest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this depth?  
**Deepest habitat: 4km subsurface sediments (terrestrial, probably marine as well)**  
**Limiting factor: temperaure (125 degrees Celcius; about 22 degrees/km)**  
**Mariana's Trench is 10.9 km, so deepest habitat is probably ~15km**  

* Based on information provided in the text your knowledge of geography what is the highest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this height? 
**Mount Everest is 8.8km**  
**There are spore-forming bacteria 57-77km in the air**  
**Limiting factor: resources, ionizing UV radiation, lack of moisture, bad weather conditions**  
**- so there are probably no microbes actually metabolizing up there, just transporting**  
**We can estimate 20 km to be conservative, so highest is 29 km**  

* Based on estimates of prokaryotic habitat limitation, what is the vertical distance of the Earth's biosphere measured in km?  
**24 km (Mount Everest to 4 km below Mariana's Trench)**

* How was annual cellular production of prokaryotes described in Table 7 column four determined? (Provide an example of the calculation)  
**The turnover time in days is converted to years. Then, the population size is divided by the turnover rate in years.**  
*(16 days)/(365 days/year) = 0.0438 years*  
*(3.6 x 10^28^ cells)/(0.0438 years) = 8.2 x 20^29^ cells/year*

* What is the relationship between carbon content, carbon assimilation efficiency and turnover rates in the upper 200m of the ocean? Why does this vary with depth in the ocean and between terrestrial and marine habitats?  
**Carbon efficiency/growth efficiency is 20% --> they used a 4 multiplier instead of 5 (how well an organism can assimilate carbon into biomass)**  
*20fg of carbon in a prokaryotic cell --> 20^-30^ Pg/cell*  
*3.6 x 10^28^ cells x 20^-30^ Pg/cell = 0.72 Pg of C in marine heterotrophs*  
*4 x 0.72 Pg C = 2.88 Pg C/year*  
*51 Pg C/year x 0.85 (85% consumed) = 43 Pg C/year*  
*(43 Pg C/year)/(2.88 Pg C/year) = 14.9/year turnovers* OR ***1 turnover every 24.5 days* **    
**Contrasting the amount of carbon that is produced and consumed in different parts of the biosphere**  

* How were the frequency numbers for four simultaneous mutations in shared genes determined for marine heterotrophs and marine autotrophs given an average mutation rate of 4 x 10^-7^ per DNA replication? (Provide an example of the calculation with units. Hint: cell and generation cancel out)  
**The mutation rate was calulcated to the power of 4 (for 4 simultaneous mutations. Then, the total number of cells was multipled by the turnover rate to calculate cells/year. That value was multiplied by the simultaneous mutation rate to give us the number of mutations per year.**  
*4 x 10^-7^ mutations/generation*   
*(4 x 10^-7^)^4^ = 2.56 x 10^-26^ mutations/generation*  
*365/16 turnovers = 22.8 turnovers/year*  
*3.6 x 10^28^ cells x 22.8 turnovers/year = 8.2 x 10^29^ cells/year*  
*(8.2 x 10^29^ cells/year) x (2.56 x 10^-26^ mutations/generation) = 2.1 x 10^4^ mutations/year --> 0.4 hours/mutation*  

* Given the large population size and high mutation rate of prokaryotic cells, what are the implications with respect to genetic diversity and adaptive potential? Are point mutations the only way in which microbial genomes diversify and adapt?  
**Since prokaryotes have high mutation rates in such a large population, they are bound to get numerous mutations that end up persisting in the population. This leads to genetic diversity as different mutations accumulate, resulting in different strains and species. Genomes can also diversify and adapt with horizontal gene transfer.**  

* What relationships can be inferred between prokaryotic abundance, diversity, and metabolic potential based on the information provided in the text?   
**The relationship between these three can be seen as cyclic. Prokaryotes are so abundant because of their rapid growth rate and ability to survive in harsh conditions due to the amount of metabolic processes they can carry out. Because they are so abundant with a rapid growth rate, this leads to genetic diversity, which results in prokaryotes that can perform many different metabolic reactions. The fact that they have so much metabolic potential allows their abundance to continue.**  

### Problem set 02   
[Falkowsi *et al* 2008](https://www.ncbi.nlm.nih.gov/pubmed/18497287)    

#### Learning objectives  
Discuss the role of microbial diversity and formation of coupled metabolism in driving global biogeochemical cycles.

#### Specific Questions  
* What are the primary geophysical and biogeochemical processes that create and sustain conditions for life on Earth? How do abiotic versus biotic processes vary with respect to matter and energy transformation and how are they interconnected?  
**- geophysical: tectonics + atmospheric photochemical processes**  
**- biogeochemical: nitrogen cycle**  
**- abiotic: acid-base reactions (transfer of protons without electrons)**  
**- biotic: redox reactions (transfer of electrons and protons)**  
**Abiotic reactions pave the way of redox reactions**  

* Why is Earth's redox state considered an emergent property?  
**- feedback between he evolution of microbial metabolic and geochemical processes create the average redox condition of the oceans and atmosphere. It is the result of a combination of processes**  
  

* How do reversible electron transfer reactions give rise to element and nutrient cycles at different ecological scales? What strategies do microbes use to overcome thermodynamic barriers to reversible electron flow?  
**- reduction of chemicals (eg. Nitrogen) fixes them in the environment - used by higher-level organisms**   
**- synergistic cooperation of microbes; different microbes have different enzymes that have thermodynamically favourable reactions in different conditions causing a cycle**  

* Using information provided in the text, describe how the nitrogen cycle partitions between different redox "niches" and microbial groups. Is there a relationship between the nitrogen cycle and climate change?  
**3 niches**  
**- in presence of O2, NH4+ can be oxidized to nitrate in a 2-stage pathway (specific group of bacteria/archaea oxidizing ammonia to NO2-, which is then oxidized to nitrate by different nitrifying bactera)**  
**- all nitrifiers use the small differences in redox potential in the oxidation reactions to reduce CO2 to organic matter**  
**- in the absense of O2, set of microbes use NO2- and NO3- as electron acceptors in the anaerobic oxidation of organic matter, which forms N2 to complete the nitrogen cycle**  
**Climate change affects the atmospheric composition, which in turn affects which set of microorganisms (which niche) will be active. This could throw off the nitrogen cycle and make parts of it more active than we are accustomed to.**  

* What is the relationship between microbial diversity and metabolic diversity and how does this relate to the discovery of new protein families from microbial community genomes?  
**Microbial diversity and metabolic diversity are positively related. There could be limitless evolutionary diversity in nature. As new protein families are discovered, it reinforces the concept that there is a large number of undiscovered species with undiscovered protein families**  

* On what basis do the authors consider microbes the guardians of metabolism?  
**The core global gene set is dispered among microbes by either vertical or horizontal gene transfer. Boutique genes have evolved due to environmental selection, and this protects the metabolic pathways these genes encode. If it goes extinct in one unit, there will be another microbial unit in which it will likely survive.**  

### Writing Assessment 01 
"Microbial life can easily live without us; we, however, cannot survive without the global catalysis and environmental transformations it provides" (Falkowski, 2008). I believe that humans would not be able to live the same quality of life they are currently living if microorganisms were to disappear from the Earth.   

Out of sight, out of mind? Microorganisms are usually not a part of the average person's daily thoughts. However, microorganisms such as bacteria and archaea play an extremely large role in our everyday lives. They aid in many bodily functions without us realizing, are present in a host of our food, and perhaps most importantly, are responsible for shaping the modern Earth's conditions. One could argue humans only exist today because of the atmospheric conditions that allowed for life on land, and subsequently, *Homo sapiens*, to evolve. The reason for those atmospheric conditions is microorganisms and the shifts in biogeochemical cycles that they cause.   

This paper will not argue that humans would immediately die without microorganisms. However, I believe that without microorganisms, humans would not be able to enjoy the same, comfortable living conditions we enjoy today. I will argue for this by explaining how microorganisms are largely responsible for the global atmospheric composition. Next, I will argue that microorganisms are able to retain pathways and maintain diversity in ways humans can not. Finally, I will state why these processes are so important for humankind.	  
  	

#### **Microorganisms are responsible for global atmospheric conditions**
The atmospheric conditions on Earth are mostly in the hands of microbial metabolism. Microorganisms are directly or indirectly responsible for almost all of the oxygen we breathe (Kasting and Siefert, 2002). The great oxygenation event occurred about 2 billion years ago and is thought to be the shift in Earth's history that allowed eukaryotic life to develop. This event can most likely be attributed to cyanobacteria, as they can live and grow both aerobically and anaerobically (Kasting and Siefert, 2002). Without this major event in Earth's history, it is certain that humans would not exist today.  

In our modern world, we often attribute the oxygen we breathe to trees and land plants as they are what we see on a daily basis. However, photosynthesis due to land plants have very little effect on atmospheric oxygen because the process is essentially reversed by the processes of respiration and decay (Kasting and Siefert, 2002). Marine photosynthesis, most of which can be attributed to bacteria and archaea (in addition to eukaryotes like algae), on the other hand, results in a net oxygen flux because "a small fraction of organic matter synthesized in the oceans is buried in sediments", resulting in the leak of most of the global oxygen in the atmosphere (Kasting and Siefert, 2002). In addition to oxygen, prokaryotic bacteria are the main organisms responsible for fixing nitrogen (that is, reducing nitrogen gas) even though they make up only about 1% of the marine biomass (Kasting and Siefert, 2002). Microorganisms have highly evolved biological machinery which allows them to work together to complete different parts of the nitrogen cycle, which will be discussed in the next section. Almost all trace gases in our modern atmosphere, such as CH4, N2O, CH3Cl, COS, and dimethyl sulfide, are almost entirely biologically produced (Kasting and Siefert, 2002).  

There is increased methane in the atmosphere today due to human activities leading to global warming. Methane gas today only has a 10-year residence time because of the reaction with the hydroxyl radical, but in anoxic times it could persist for 10,000 years (Kasting and Siefert, 2002). The gas was produced by methanogens, creating a methane haze which kept the Earth from freezing. Without the methane production in that era of Earth's history, the Earth would most likely be frozen. Archaea and bacteria can be said to have determined the basic atmospheric conditions since the origin of life (Kasting and Siefert, 2002).  


#### **Microorganisms can uniquely maintain diversity and retain pathways**  
Microorganisms are without a doubt the most abundant organisms on the planet. This abundance, paired with a rapid growth rate, is highly relevant in terms of its support in microorganisms' abilities to create and maintain genetic diversity. Their high turnover rate makes them especially susceptible to mutation accumulation which directly contributes to genetic diversity (Whitman *et al.*, 1998). This allows for development of crucial pathways and maintenance of redundant pathways. Although there is enormous genetic diversity, the core genes for major reduction-oxidation (redox) reactions have been preserved and are stable. These genes are essential for life and biogeochemical pathways (Falkowski, 2008).  

The term emergent property is a popular one in the discussion of microorganisms. An emergent property refers to one that is not present in individuals but comes as a result of individuals working together. Many processes carried out by microorganisms are done by distributing different components of the pathways to different organisms. This specific trait of microorganisms is crucial in preventing pathways from dying out. In the event of a species going extinct, only one component of a pathway would be missing. As mentioned earlier, microbial pathways are often redundant and so a missing cog in the system could be easily replaced. The genes that were created and coevolved with biogeochemical cycles were passed between microorganisms primarily through horizontal gene transfer (Falkowski, 2008). If an operational taxonomic unit experiences a perturbation and doesn't survive, that unit could go extinct; however, the core machinery would survive (Falkowski, 2008).  

Without biogeochemical cycles, the planet would reach an equilibrium, resulting in a slow depletion of substrates (Falkowski, 2008). Along with geochemical processes such as abiotic methane oxidation and diagenesis, microbes are responsible for maintaining the Earth's atmospheric composition with their biogeochemical metabolism (Falkowski, 2008). The enzymes that are necessary for these biogeochemical processes are distributed amongst different microorganisms. For example, the different steps of the nitrogen cycle are carried out by different microorganisms who, together, complete the cycle. Because of the reactions carried out by microorganisms working together by distributing pathways, it is ensured that the Earth's atmosphere continues to proceed as a cycle. Therefore, it can be stated that the redox state of the global atmosphere is an emergent property produced by microorganisms.  

One reason that prokaryotes are abundant is because of their ability to survive in harsh conditions. They can survive in these harsh conditions because of the various metabolic processes they can carry out, giving them the ability to thrive in vastly different environments. Due to their abundance and high growth rate, they have enormous genetic diversity, which results in more prokaryotes that can perform many different metabolic reactions as well as the conservation of necessary pathways. This pattern of prokaryotic metabolic potential ensures that pathways continue to evolve as needed, and that redundancy exists so pathways are hardy and can survive over time.  


#### **Humans could not survive without microbial processes**
All of the processes described above are essential for human life. Microbes keep atmospheric oxygen at appropriate levels and convert nitrogen into usable form for other living organisms. Humans tend to believe they are the drivers of the world and they are the sole reason the planet is the way it is. Essentially, they believe they are self-sufficient in many ways. However, the processes that microorganisms perform at the scale that they do is non-replicable by non-biological (in terms of machines) or biological (in terms of other organisms) means. Elemental cycles are often studied in isolation, but it must be taken into account that they have coevolved and they all influence each other (Falkowski, 2008). When studying them, it is unknown how long they take to develop but it is known that photosynthesis was one of the last biogeochemical cycles to emerge (Falkowski, 2008). It would require a much deeper understanding of the cycles in order to replicate them without microorganisms.   

In addition, humans do not share the same level of abundance, growth rate, and evolution rate as prokaryotes. Humans also do not combine efforts in order to complete metabolic pathways; we complete our required metabolic pathways by ourselves. Therefore, it would be impossible for us to evolve, nonetheless maintain valuable enzymes required for human metabolism in the same way that microorganisms do. Furthermore, the efficiency of microorganisms would need to be considered as well, since they carry out these essential biogeochemical processes in relation with each other which allows them to streamline their metabolism.  

While I do not believe humans would cease to exist, the quality of life without microorganisms would drastically decrease. There would be a drastic reduction in oxygen, causing carbon dioxide levels to increase and thereby increasing the effects of man-made global warming. It is also worth noting that our gut microbiota quietly carries out a vast number of processes in our own body throughout the day, allowing us to live comfortably. Suffice to say, a world without microorganisms would be an enormous shift.  


#### **Conclusion**
Prokaryotes are the drivers of this planet. They are responsible for the majority of the global atmospheric conditions, have the ability to maintain diversity and important pathways in ways that humans can not, and carry out processes that are essential for and cannot be replicated by humans. A more in-depth analysis could reveal how long humans could survive and how different their quality of life would be. It remains to be seen to what extent humans could maintain their survival without microorganisms.  

#### **References**
Falkowski, P. G. *et al.* (2008). [The Microbial Engines That Drive Earth's Biogeochemical Cycles.](https://www.ncbi.nlm.nih.gov/pubmed/18497287) Science, 320, 1034-1039.  

Kasting, J. F. and Siefert, J. L. (2002). [Life and the Evolution of Earth's Atmosphere.](https://www.ncbi.nlm.nih.gov/pubmed/12004117) Science, 296, 1066-1068.  

Whitman, W.B., Coleman, D.C., and Wiebe, W. J. (1998). [Prokaryotes: the unseen majority.](https://www.ncbi.nlm.nih.gov/pubmed/9618454) National Academy of Science, 95, 6578-6583.   

### Module 01 references
Achenback J. 2012. [Spaceship Earth: A new view of environmentalism.](https://www.washingtonpost.com/national/health-science/spaceship-earth-a-new-view-of-environmentalism/2011/12/29/gIQAZhH6WP_story.html?utm_term=.4d3aeac98db8) The Washington Post.  

Canfield DE, Glazer AN, and Falkowski PG. 2010. [The Evolution and Future of Earth's Nitrogen Cycle.](https://www.ncbi.nlm.nih.gov/pubmed/20929768) Science. 330(6001):192-196.  

Falkowski PG, Fenchel T, and Delong EF. 2008. [The Microbial Engines That Drive Earth's Biogeochemical Cycles.](https://www.ncbi.nlm.nih.gov/pubmed/18497287) Science. 320(5879):1034-1039.  

Falkowski PG *et al*. 2000. [The Global Carbon Cycle: A Test of Our Knowledge of Earth as a System.](http://science.sciencemag.org/content/290/5490/291) Science. 290(5490):291-296.

Kallmeyer J *et al.*. 2012. [Global distribution of microbial abundance and biomass in subseafloor sediment.](https://www.ncbi.nlm.nih.gov/pubmed/22927371) Proc Natl Acad Sci USA. 109(40):16213-16216.   

Kasting JF and Siefert JL. 2002. [Life and the Evolution of Earth's Atmosphere.](http://science.sciencemag.org/content/296/5570/1066.long) Science. 295(5570):1066-1068.  

Leopold A. 1949. [The Land Ethic.](http://rintintin.colorado.edu/~vancecd/phil3140/Leopold.pdf) Location: Publisher. *A Sand County Almanac*. 

Mooney C. 2016. [Scientists say humans have now brought on an entirely new geologic epoch.](https://www.washingtonpost.com/news/energy-environment/wp/2016/01/07/scientists-say-humans-have-now-brought-on-an-entirely-new-geologic-epoch/?utm_term=.fe0893313ea1) The Washington Post.  

Nisbet EG and Sleep NH. 2001. [The habitat and nature of early life.](https://www.nature.com/articles/35059210) Nature. 409:1083-1091.   

Schrag DP. 2012. [Geobiology of the Anthropocene](https://onlinelibrary.wiley.com/doi/10.1002/9781118280874.ch22). Wiley Online Library.  

Rockstrom J *et al.* 2009. [A safe operating space for humanity.](https://www.nature.com/articles/461472a) Nature. 461:472-475.  

Waters CN *et al.* 2016. [The Anthropocene is functionally and stratigraphically distinct from the Holocene](http://science.sciencemag.org/content/351/6269/aad2622). Science. 351(6269):aad2622.  

Whitman WB, Coleman DC, and Wiebe WJ. 1998. Prokaryotes: The unseen majority. Proc Natl Acad Sci USA. 95(12):6578-6583.  [PMC33863](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)  



## Data science Friday  

### Installation check
![Figure 1.](images/DS_Assignment1.1.png){width=50%}  

![Figure 2.](images/DS_Assignment1.2.png){width=50%}  

![Figure 3.](images/DS_Assignment1.3.png){width=50%}  

### Portfolio repo setup
1) To create my portfolio:  
  `mkdir MICB245_portfolio`

2) To initialize my portfolio:  
a) Created a repository called MICB425_portfolio on GitHub GUI  
b) Return to terminal:  
  `cd ~/Documents/MICB425_portfolio`  
  `git init`  
  
3) To push my portfolio to GitHub:  
  `git add .`  
  `git commit -m "message"`  
  `git status`   
  `git push`


### RMarkdown pretty html challenge
See .Rmd and .html files "DS_assignment3_19163147" in portfolio repo

### Assignment 4  
See R script and .html files "DS_assignment4_19163147" in portfolio repo


# Module 2 
## Metagenomics: Genomic Analysis of Microbial Communities  
### Evidence worksheet 04 
[Martinez *et al.* 2007](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838496/)  

#### Learning objectives  
Discuss	the	relationship	between	microbial	community	structure	and	metabolic	diversity. Evaluate	common	methods	for	studying	the	diversity	of	microbial	communities. Recognize basic	design	elements	in	metagenomic	workflows.  

#### Specific Questions:
* What were the main questions being asked?  
**How can we decribe PR photosystem genetics and biochemistry?**  
**Further characterize PR photosystem structure and function**  


* What were the primary methodological approaches used?  
**Functionally surveyed marine picoplankton large-insert genomic library for recombinant clones expressing PR photosystems in vivo**  
**Exploited transient increases in vector copy number that significantly enhanced the sensitivity of phenotypic detection**  

* Summarize the main results or findings.   
**1) Two genetically distinct recombinants (identified by their orange pigmentation) expressed a cluster of genes encoding a complete PR-based photosystem**  
**2) Heterologous expression of 6 genes (5 for photopigment biosynthetic proteins and 1 encoding a PR) generated a fully functional PR photosystem that enabled photophosphorylation in recombinant *E. coli* cells exposed to light**  
**3) A single genetic event can result in the acquisition of phototrophic capabilities in an otherwise chemoorganotrophic microorganism**  
**4) Explains the ubiquity of PR photosystems among diverse microbial taxa**  
**5) Took dna from environment, programmed heterologous host, looked for particular phenotype, discovered minimal set of genes required to express that phenotype**  

* Do new questions arise from the results?  
**How can downstream genetic and biochemical characterization using bioinformatics occur? What pathway does the PR photosystem use to utilize light to biochemical energy conversion?**  

* Were there any specific challenges or	advantages in	understanding the	 paper (e.g.	 did	 the	authors	 provide	 sufficient background information to understand experimental logic, were methods	explained	adequately,	were	any	specific	assumptions	made,	were	conclusions	justified	based	on	the	evidence,	were	the	figures	or tables useful and	easy to understand)?  
**Unsure of the purpose of inserting either one or multiple copies of the fosmid**  

### Problem set 03   
#### Learning objectives: 
Specific emphasis should be placed on the process used to find the answer. Be as comprehensive as possible e.g. provide URLs for web sources, literature citations, etc.  


#### Specific Questions:
* How many prokaryotic divisions have been described and how many have no cultured representatives (microbial dark matter)?  
**2016: 89 bacterial phyla, 20 archael phyla (via 16S rRNA databases). But could be up to 1500 bacterial phyla, as there are microbes that libe in the "shadow biosphere"**  
**2003: 26 of 52 major bacterial phyla have been cultivated**  

* How many metagenome sequencing projects are currently available in the public domain and what types of environments are they sourced from?  
**According to EBI Metagenomics, there are 110,217 data sets which consists of 10,653 metagenomes. They were sourced from multiple environments, including soil, humans, marine, freshwater, plants, and grassland.**  

* What types of on-line resources are available for warehousing and/or analyzing environmental sequence information (provide names, URLS and applications)?  
*Shotgun Metagenomics*  
**Assembly - Euler**  
**Binning - S-GCOM**  
**Annotation - KEGG**  
**Analysis pipelines - Megan 5**  
*Marker Gene Metagenomics*  
**Standalone software - OTUbase**  
**Analysis pipelines - SILVA (gold standard)**  
**Denoising - Amplicon Noise**  
**Databases - RDP (also good)**  
*Data Warehousing - Databases*  
**IMG/M, MG-RAST, NCBI/EBI which will give you sequences to feed into the aforementioned softwares**  

* What is the difference between phylogenetic and functional gene anchors and how can they be used in metagenome analysis?  
**Phylogenetic: vertigal gene transfer, carry phylogenetic information allowing tree reconstruction, taxonomix, ideally single-copy**  
**Functional: more horizontal gene transfer, identify specific biogeochemical functions associated with measurable effects, not as useful for phylogeny**  

* What is metagenomic sequence binning? What types of algorithmic approaches are used to produce sequence bins? What are some risks and opportunities associated with using sequence bins for metabolic reconstruction of uncultivated microorganisms?  
**The process of grouping of sequences that come from a single genome**  
**Types of algorithms: align sequences to database, group to each other based in DNA characteristics (GC content, codon usage)**  
**Risks: incomplete coverage of genome sequence, contamination from different phylogeny**  

* Is there an alternative to metagenomic shotgun sequencing that can be used to access the metabolic potential of uncultivated microorganisms? What are some risks and opportunities associated with this alternative?  
**Functional screens (biochmical assays, etc.), 3rd generation sequencing (nanopore), single cell sequencing, FISH probe**  

### Module 02 references
Madsen EL. 2005. [Identifying microorganisms responsible for ecologically significant biogeochemical processes](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838496/). Nature Reviews Microbiology. 104(13):5590-5595.

Martinwz A, Bradley AS, Waldbauer JR, Summons RE, and DeLong EF. 2007. [Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host](https://www.nature.com/articles/nrmicro1151). Proc Natl Acad Sci USA. 3:439-446.  

Taupp M, Mewis K, and Hallam SJ. [The art and design of functional metagenomic screens](https://www.ncbi.nlm.nih.gov/pubmed/21440432). Curr Opin Biotechnol. 22(3):465-472.

Wooley JC, Godzik A, and Friedberg I. 2010. [A Primer on Metagenomics](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000667). PLoS Computational Biology. 6(2).

# Module 3  
## Microbial Species Concepts  
### Evidence worksheet 05
[Welch RA *et al.* 2002](https://www.ncbi.nlm.nih.gov/pubmed/12471157)  

#### Learning objectives part 1: 
Evaluate the concept of microbial species based on environmental surveys and cultivation studies. Explain the relationship between microdiversity, genomic diversity and metabolic potential. Comment on the forces mediating divergence and cohesion in natural microbial
communities


#### Specific Questions:
* What were the main questions being asked?  
**The different genes involved in E. coli pathogenicity and the difference between different strains of E. coli**  
 

* What were the primary methodological approaches used?  
**Isolate from a hospital patient**  
**Random clones sequenced using dye-terminator chemistry and data collected on ABI sequencers**  
**Sequence data assembled by SEQMANII**  
**Whole genome XhoI optical map for ordering of contigs and providing physical map of whole genome**  
**Annotations using MAGPIE**  


* Summarize the main results or findings. 
**Only 39% of combined sets of proteins between a uropathic, enterohemorrhagic, and laboratory strain E. coli are common to all 3 strains**  
**No virulence plasmid found; 5 cryptic prophages; 70% of ORFs identified as unique to ED or MF are now replaced with new genes which are specific to CFT073**  
**Analysis done on CFT073 on the basis that codon usage would defer between the backbone and the islands; 52 of 61 island ORFs had significantly different frequency distribution than from the backbone**  


* Do new questions arise from the results?  
**What criteria should we use to differntiate species and their diversity?**  
**How diverse are the different types of microbes in different ecotypes even when they're related?**  


* Were there any specific challenges or	advantages in	understanding the	 paper (e.g.	 did	 the	authors	 provide	 sufficient background information to understand experimental logic, were methods	explained	adequately,	were	any	specific	assumptions	made,	were	conclusions	justified	based	on	the	evidence,	were	the	figures	or tables useful and	easy to understand)?	
**The images were difficult to understand and weren't clearly explained, so it was a bit hard to follow**  
**Background material was sufficient to understanding the context of the paper**  

#### Learning objectives part 2:  
Comment	on	 the	creative	 tension	between	gene	loss,	duplication	and	acquisition	as	it	
relates	to	microbial	genome	evolution. Identify	common	molecular	signatures	used	to	infer	genomic	identity	and	cohesion. Differentiate	between	mobile	elements	and	different	modes	of	gene	transfer.  

#### Specific Questions:  
Based	on	your	reading	and	discussion	notes,	explain	the	meaning	and content	of	the	following	figure	derived	from	the	comparative	genomic	analysis	of	three E.	coli genomes	by	Welch	et	al.	
Remember	 that	 CFT073	 is	 a	 uropathogenic	 strain	 and that	 EDL933	 is	 an	 enterohemorrhagic	strain.	 Explain	 how	 this	 study	 relates	 to	 your understanding	 of	 ecotype	 diversity. Provide	 a	definition	of	ecotype	in	the	context	of	the human	body.	Explain	why	certain	subsets	of	genes	in	CFT073	 provide	 adaptive	 traits under	 your	 ecological	 model and	 speculate	 on	 their	 mode	 of	vertical	descent	or	gene transfer.  

**The figure displays the size and location of islands found in both the CFT073 and EDL933 strains. These shared islands encode putative iron-uptake systems, a complex set of potential fatty acid biosynthetic enzymes, several adhesins, and phosphotransferase system and ATP-binding cassette (ABC)-type transport systems. It's interesting that despite being inserted at the same location in the backbone of both strains that the two islands may have completely different content. Ecotype diversity refers to the genotypic and phenotypic differences within populations of a single species. In this study, we see that CFT073 and EDL933 are found in two different locations of the human body. The environmental differences in the two locations pose different challenges for the two strains, causing CFT073 to express a different subset of genes than EDL933.**   


### Problem set 04
See .Rmd and .html files "Problem_set_04_19163147" in portfolio repo

### Writing Assessment 03 
A eukaryotic species is relatively simple to define, as the species criteria (for most eukaryotes) is the ability to sexually reproduce within that species and no other species. For prokaryotes, however, the concept of species becomes more difficult to define, as many of them reproduce asexually. Currently, microorganisms are classified by their similarities in size, shape, and most importantly, genes. The use of 16S rRNA to phylogenetically classify species has greatly increased our estimates of microbial diversity (Riley and Lizotte-Waniewski, 2010). However, horizontal gene transfer (HGT) plays a largely underestimated role in evolution. Since certain genes can be transferred between completely different species, the microbial phylogenetic tree becomes more of a network. This is one of the reasons defining a microbial species is increasingly challenging. This paper will discuss the challenges in defining microbial species, with an emphasis on how HGT complicates the matter. In addition, it will analyze how HGT influences the maintenance of biogeochemical processes through time and extinctions. Finally, I will argue that it is necessary to have a clear definition of a microbial species so that an accurate representation of biological diversity can be maintained and understood.  

#### **Challenges in defining microbial species** 
There are countless challenges that arise when attempting to define a microbial species. Due to their small size and morphological simplicity of prokaryotes, it is not feasible to use appearance as a species classifier as it is used for higher eukaryotes (Rossello-Mora and Amann, 2006). In addition, since there are very few useful fossil records, the ancient history of microbial evolution is difficult to track. Several different concepts of how to distinguish species have been proposed (Wilkins, 2006). One of these is based on mechanisms of genetic exchange and recombination similar to sexual reproduction which allows for maintenance of a core genome. Another concept proposes that species are merely ecotypes, and species are defined by where they are found. A third option states species concepts are too conventional to capture the essence of microbial life, and no definition will apply to every case (Wilkins, 2006).  

16S rRNA is the most common determining factor used for microbial species classifications (Rossello-Mora and Amann, 2006). The problem with this technique is that it assumes that gene transfer has not occurred between rRNA genes and the amount of dissimilarity between rRNA sequences is representative of the dissimilarity of genomes (Rossello-Mora and Amann, 2006). This method is not as effective when examining closely related species, making it difficult to use to identify new species if they are similar to existing species in terms of their 16S rRNA. There are examples of different species with identical 16S sequences, and small differences in 16S sequences within members of the same species (Rossello-Mora and Amann, 2006). Therefore, 16S is not a completely accurate measure on its own for species definitions.  

Since prokaryotes do not reproduce sexually, the main driver of genetic diversity is mutations. Moreover, prokaryotes have evolved HGT systems in which different species can exchange genetic information. The problem with 16S rRNA sequencing is that it only relies on one gene, and HGT may play a large role in evolution that is not accounted for. However, HGT may also result in vastly different species being classified as phylogenetically related, even though there is not a phylogenetic relationship between the two, as HGT violates species boundaries as opposed to preserving them (Ereshefsky, 2010). Prokaryotes can also undergo homologous recombination with species that are up to 25% divergent in the sequences of homologous genes and can express plasmid genes from extremely divergent species (Rossello-Mora and Amann, 2006). This complicates species boundaries because vastly different species can share very similar genes, which makes the use of genes to identify species even more complex.  

#### **Influence of HGT on biogeochemical process maintenance**
Microorganisms are the drivers of Earth's biogeochemical cycles, and horizontal gene transfer has enabled core genes to persist. While the Earth's atmosphere has changed drastically, there exists a core set of microbial genes that still remain today (Falkowski *et al.*, 2008). HGT allows for highly advantageous genetic information to be shared in changing environmental conditions (Drudge and Warren, 2012). These genes are responsible for coding enzymes responsible for the major biogeochemical cycles that sustain life. One example of this is sulfite reductases which are found in a variety of microorganisms such as sulfate-reducing proteobacteria, Gram-positive bacteria, and Archaea (Falkowski *et al.*, 2008). In addition to genes, entire metabolic pathways have been transferred via HGT from microbe to microbe. For example, the entire anoxygenic photosynthetic apparatus appears to have been transferred between phototropic proteobacteria, and similar transfers could have occurred during the establishment of oxygenic photosynthesis (Nagashima *et al.*, 1997). HGT has allowed for the distribution of different components of biogeochemical pathways to different organisms. This trait is crucial in preventing pathways from dying out.   

HGT allows for the presence of redundant pathways, so that if one species went extinct, there would be numerous others that could carry out the same metabolic processes. Therefore, while certain microorganisms could be wiped out, the core machinery would survive. While many extinction events have occured over Earth's history, microorganisms have been able to maintain their essential genes that are necessary for the survival of life. The cause of the retention of horizontally transferred genes is likely nutritional selective pressures, which allows different organisms in different environments to preserve these genes (Falkowski *et al.*, 2008).   

#### **Necessity of microbial species definition**
I believe it is important to have a clear definition for what is considered a microbial species. Without a recognized definition, we would have no basis to judge whether species diversity was increasing or decreasing over time. In addition, we wouldn't be able to track evolutionary change, namely because if we don't have definitions for species, we cannot know when new species arise. In addition, if a universal definition for species isn't used among researchers, it will result in inconsistencies when a claim is made about a new species. Furthermore, inconsistency is species definition would be troublesome for pathologists when trying to identify the cause of an infection. If two related species are broadly claimed to be the same species due to inconsistent species definitions, then an inappropriate treatment may be recommended.    

Sogin et al. estimate that the complexity of microbial life is 100 times greater than current estimates (2006). While this is largely due to incomplete sampling of the vast ecosystems present on Earth, it could also be an artifact of unclear species definitions defining multiple different species as the same. This poses a problem for ecological microbiologists, as these different kinds of microorganisms may be members of different biogeochemical pathways. Without a clear species definition, it may be difficult to fully understand the workings of microbial metabolic processes and discover new processes.   

However, it may be possible for some leniency to be allowed with the species definition. Prokaryotes have many different modes of evolution, so a strict species definition may not encompass these varying methods. Due to horizontal gene transfer, the boundary that separates species is blurred. Similar to how defining a eukaryotic species is different than a prokaryotic species due to the differences in the reproduction, perhaps there could be some differences in the definitions of different prokaryotic groups based on their methods of reproduction. This may allow for consistency based on organisms who use horizontal gene transfer and other forms of genetic exchange, and organisms who primary reproduce by binary fission and evolve through mutations.   

#### **Conclusion**
Defining prokaryotic species is one of the major challenges facing the microbiological world today. Without a clear species definition, inconsistencies among researchers, ecologists, and pathologists could arise, hindering their ability to move forward in their fields. Horizontal gene transfer is a large factor in microbial genetic exchange, even between unrelated microbial species. While this process allows for the conservation of core genes responsible for the regulation of Earth's biogeochemical processes, it makes it increasingly difficult to identify species and differentiate them. Ochman *et al.* have looked at the extent of gene transfer that occurs within species and between species, and a more in-depth analysis could look further at the major processes that have been conserved due to horizontal gene transfer. It remains to be seen how HGT will affect the ever-changing definition of a microbial species.  

#### **References**
Drudge, C.N. and Warren, L.A. (2012). [Prokaryotic Horizontal Gene Transfer in Freshwaer Lakes: Implications of Dynamic Biogeochemical Zonation.](http://file.scirp.org/Html/5-6701657_25690.htm) Journal of Environmental Protection, 3(12): 1634-1654.  

Ereshefksy, M. (2010). [Microbiology and the species problem.](https://phil.ucalgary.ca/marcereshefsky/sites/phil.ucalgary.ca.marcereshefsky/files/microbiology_and_species.pdf) Biol Philos, 25: 553-568. 

Falkowski, P.G. *et al.* (2008). [The Microbial Engines That Drive Earth's Biogeochemical Cycles.](https://www.ncbi.nlm.nih.gov/pubmed/18497287) Science, 320, 1034-1039.  


Nagashami, K.V.P. *et al.* (1997). [Horizontal transfer of genes coding for the photosynthetic reaction centers of purple bacteria.](https://link.springer.com/article/10.1007%2FPL00006212) Journal of Molecular Evolution, 45(2): 131-136. 

Ochman, H. *et al.* (2000). [Lateral gene transfer and the nature of bacterial innovation.](https://www.ncbi.nlm.nih.gov/pubmed/10830951) Nature, 405(6784): 299-304.

Riley, M.A. and Lizotte-Waniewski M. (2009). [Population Genomics and the Bacterial Species Concept.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2842946/) Methods Mol Bol, 532: 366-377.  

Rossello-Mora, R. and Amann, R. (2006). [The species concept for prokaryotes.](https://www.ncbi.nlm.nih.gov/pubmed/11152940) FEMS Microbiology Reviews, 25(1).  

Sogin, M.L. *et al.* (2006). [Microbial diversity in the deep sea and the underexplored "rare biosphere".](https://www.ncbi.nlm.nih.gov/pubmed/16880384) PNAS, 103(32): 12115-12120.  

Wilkins, J.S. (2006). [The Concept and Causes of Microbial Species.](http://philsci-archive.pitt.edu/3426/) Hist. Phil. Life Sci., 28: 389-408.  


### Module 03 references
Callahan BJ, McMurdie PJ, and Holmes SP. 2017. [Exact sequence variants should replace operational taxonomic units in marker-gene data analysis](https://www.ncbi.nlm.nih.gov/pubmed/28731476). ISME J. 11(12):2639-2643.  

Codero OX *et al.* 2012. [Public good dynamics drive evolution of iron acquisition strategies in natural bacterioplankton populations.](https://www.ncbi.nlm.nih.gov/pubmed/23169633) Proc Natl Acad Sci USA. 109(49):20059-20064.  

Gaudet AD, Ramer LM, Nakonechny J, Cragg JJ, and Ramer MS. 2010. [Small-Group Learning in an Upper-Level University Biology Class Enhances Academic Performance and Student Attitudes Toward Group Work](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0015821). PLoS.   

Giovannoni SJ. 2012. [Vitamins in the sea.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3435215/) Proc Natl Acad Sci USA. 109(35):13888-13889.  

Hallam SJ, Torres-Beltran M, and Hawley AK. 2017. [Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone](https://www.nature.com/articles/sdata2017158). Scientific Data. 4(170158).

Hawley AK *et al*. 2017. [A compendium of multi-omic sequence information from the Saanich Inlet water column](https://www.nature.com/articles/sdata2017160). Scientific Data. 4(170160).  

Kunin V *et al*. 2010. [Wrinkles in the rare biosphere: pyrosequencing errors can lead to artificial inflation of diversity estimates.](https://www.ncbi.nlm.nih.gov/pubmed/19725865) Environmental Microbiology. 12(1):118-123.  

Lundin D *et al*. 2012. [Which sequencing depth is sufficient to describe patterns in bacterial alpha- and beta-diversity?](https://www.ncbi.nlm.nih.gov/pubmed/23760801) Environmental Microbiology Rep. 4(3):367-372.   

Morris JJ, Lenski RE, and Zinser ER. 2012. [The Black Queen Hypothesis: Evolution of Dependencies through Adaptive Gene Loss](http://mbio.asm.org/content/3/2/e00036-12.abstract). mBio. 3(2).  

Sogin ML *et al.* 2006. [Microbial diversity in the deep sea and the underexplored "rare biosphere".](https://www.ncbi.nlm.nih.gov/pubmed/16880384) Proc Natl Acad Sci USA. 103(32):12115-12120.   

Thompson JR *et al.* 2005. [Genotypic diversity within a natural coastal bacterioplankton population.](https://www.ncbi.nlm.nih.gov/pubmed/15731455) Science. 307(5713):1311-1313.  

Torres-Beltran M *et al.* 2017. [A compendium of geochemical information from the Saanich Inlet water column.](https://www.nature.com/articles/sdata2017159) Scientific Data. 4(170159).

Welch RA *et al.* 2002. [Extensive mosaic structure revealed by the complete genome sequence of uropathogenic Escherichia coli](https://www.ncbi.nlm.nih.gov/pubmed/12471157). Proc Natl Acad Sci USA. 99(26):17020-17024.  
