---
title: "19163147_MICB425_portfolio"
author: "Jaspreet Gill (19163147)"
date: "version `r format(Sys.time(), '%B %d, %Y')`"
output:
   html_document: 
    toc: yes 
---
# Module 01
    
## Data science Friday  

### Installation check
![Figure 1.](images/DS_Assignment1.1.png){width=50%}  

![Figure 2.](images/DS_Assignment1.2.png){width=50%}  

![Figure 3.](images/DS_Assignment1.3.png){width=50%}  

### Portfolio repo setup
1) To create my portfolio:  
  `mkdir MICB245_portfolio`

2) To initialize my portfolio:  
a) Created a repository called MICB425_portfolio on GitHub GUI  
b) Return to terminal:  
  `cd ~/Documents/MICB425_portfolio`  
  `git init`  
  
3) To push my portfolio to GitHub:  
  `git add .`  
  `git commit -m "message"`  
  `git status`   
  `git push`


### RMarkdown pretty html challenge
See .Rmd and .html files "DS_assignment3_19163147" in portfolio repo

### Assignment 4  
See R script and .html files "DS_assignment4_19163147" in portfolio repo

## Origins and Earth Systems
### Evidence worksheet 01 
[Whitman *et al* 1998](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)

#### Learning objectives
Describe the numerical abundance of microbial life in relation to ecology and biogeochemistry of Earth systems. 

#### Specific questions
* What were the main questions being asked?  
**How many prokaryotes are present? How much carbon do they produce?**  

* What were the primary methodological approaches used?  
**In aquatic environments, they measured cellular density in cells/mL. In soil, they estimated abundance from unpublished field studies. For subsurface environments, they used two different methods. They either estimated from groundwater data or calculated number of prokaroytes based on the assumptions that average porosity is 3% as well as the average estimated volume of a prokaryotic cell.**  

* Summarize the main results or findings.  
**The total number of prokaryotes is 4-6 x 10^30^. The total cellular carbon on Earth is 350-550 Pg (so prokaryotic carbon is 60-100% of the estimated carbon in plants) and there is 10-fold more nitrogen and phosphorus in prokaryotes than plants. Most prokaryotes are found in open ocea, soil, and oceanic/terrestrial subsurfaces. Prokakryotic cellular production rate is the highest in the open ocean, and their large population size mixed with rapid growth allows for vast genetic diversity.**  

* Do new questions arise from the results?  
**How does the mutation rate affect global cycles? How does turnover rate correlate with mutation rates? How many prokaryotes are in the less-explored areas that were briefly mentioned in the paper?**  

* Were there any specific challenges or advantages in understanding the paper (*e.g.* did the authors provide sufficient background information to understand experimental logic, were methods explained adequately, were any specific assumptions made, were conclusions justified based on the evidence, were the figures or tables useful and easy to understand)?  
**They presented enough evidence in the paper to supplement the conclusion. The methods could have been a bit more descriptive, especially with the calculations. There could have been more justification in how the calculations were made.**

### Evidence worksheet 02   
[Kasting & Siefert 2002](http://science.sciencemag.org/content/296/5570/1066.long)  

#### Learning objectives  
Comment on the emergence of microbial life and the evolution of Earth systems
    
#### Specific Questions   
* Indicate the key events in the evolution of Earth systems at each approximate moment in the time series. If times need to be adjusted or added to the timeline to fully account for the development of Earth systems, please do so. 

    + 4.6 billion years ago: solar system formed, inner planets had water --> high CO2, high water vapour pressure, high temperatures    
    
    + 4.5 billion years ago: earth hit by an object causing moon to form, earth slightly tilted, night and day cycles created habitats    
    
    + 3.8 billion years ago: meteorites halted - sea water chemistry stabilized (sulfate reduction), early methanogensis  
    
    + 3.5 billion years ago: real evidence of life from carbon isotopes from ocean sedimentary rocks; rubisco - carbon fixation, photosynthesis  
    
    + 3 billion years ago: year-round ice forming at North Pole
    
    + 2.7 billion years ago: possible eukaryotes   
    
    + 2.2 billion years ago: rocks recognized as red beds - oxidation involved    
    + 2.1 billion years ago: appreciable oxygen, switch between mitochondria and chloroplast, increased complexity of life  
    
    + 1.3 billion years ago: first plants appear  
    
    + 550 million years ago: Cambrian explosion, most major animal phyla appeared - rapid expansion of life  
    
    + 550,000 years ago: Archaic humans appeared (*Homo neanderthalensis*)  
    
    + 200,000 years ago: Homo sapiens appear  


* Describe the dominant physical and chemical characteristics of Earth systems at the following waypoints:  

    + Hadean: solar system forming within large cloud of gas around sun, earth initially molten, formation of moons, Heavy Bombardment  
    
    + Archean: no continents, Earth's crust cooled, vast amounts of CO2 in atmosphere, low levels of sunlight so low heat, low oxygen levels  
    
    + Precambrian: major volcanic events altering atmospheric composition; hit by large object causing moon to form  
    
    + Proterozoic: first known glaciations, ending with Snowball Earth (entirely frozen)  
    
    + Phanerozoic: continents drifted apart, eventually connected as Pangaea and then split up again; most biological evolution
    

### Evidence worksheet 03  
[Rockstrom *et al*](https://www.nature.com/articles/461472a)  

#### Learning objectives: 
Evaluate human impacts on the ecology and biogeochemistry of Earth systems.  

#### Specific Questions:
* What were the main questions being asked?  
**How can we redfine preconditions for human development?**  
**What are the consequences of crossing these boundaries?**  

* What were the primary methodological approaches used?  
**Proposed planetary boundaries to meet challenge of maintaining Holocene state which define the safe operating space for humanity**  
**Many thresholds defined by critical value for one or more controlled variables**  
**In general, planetary boundaries are values for control variables that are either at a 'safe' distance from thresholds (for processes with evidence of threshold behaviour) or at dangerous levels (for processes without evidence of thresholds)**  


* Summarize the main results or findings.  
**9 processes:**  
**- climate change**  
**- ocean acidification**  
**- stratospheric ozone depletion**  
**- biogeochemical flow boundaries (interference with nitrogen and phosphorus cycles)**  
**- global freshwater use**  
**- change in land use**  
**- biodiversity loss (terrestrial and marine)**  
**- atmospheric aerosol loading (not yet quantified)**  
**- chemical pollution (not yet quantified)**  
**Three systems (rate of biodiversity loss, climate change, human interference with the nitrogen cycle) have already exceeded their boundaries.**  


* Do new questions arise from the results?  
**How do we define the boundaries for atmospheric aerosol and chemical pollution?**  
**Can the boundaries that have been crossed be reversed?**  
**To what extent are the boundaries linked?**  
**How long does it take to cause significant environmental change?**  

* Were there any specific challenges or	advantages in	understanding the	 paper (e.g.	 did	 the	authors	 provide	 sufficient background information to understand experimental logic, were methods	explained	adequately,	were	any	specific	assumptions	made,	were	conclusions	justified	based	on	the	evidence,	were	the	figures	or tables useful and	easy to understand)?  
**- units of biodiversity loss: why not just number of species/year? why number of species/million species/year?**  
**- explanation for why atmospheric aerosol/chemical pollution is not quantified, or perhaps an estimate would've been helpful**  


### Problem set 01
[Whitman *et al* 1998](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)  

#### Learning objectives  
Describe the numerical abundance of microbial life in relation to the ecology and biogeochemistry of Earth systems.

#### Specific questions  
* What are the primary prokaryotic habitats on Earth and how do they vary with respect to their capacity to support life? Provide a breakdown of total cell abundance for each primary habitat from the tables provided in the text.  
**Aquatic: 1.181 x 10^29^ cells**  
**Soil: 2.556 x 10^29^ cells**    
**Subsurface: 3.8 x 10^30^ cells**   

* What is the estimated prokaryotic cell abundance in the upper 200 m of the ocean and what fraction of this biomass is represented by marine cyanobacterium including Prochlorococcus? What is the significance of this ratio with respect to carbon cycling in the ocean and the atmospheric composition of the Earth?  
**Estimated cell abundance = 360 x 10^26^**  
***Prochlorococcus* = 8% --> 28.8 x 10^26^**  
**Significance for carbon cycling = 8% of upper 200m can photosynthesize --> upper ocean is ideal place for them to be so they can access sunlight**  

* What is the difference between an autotroph, heterotroph, and a lithotroph based on information provided in the text?  
**Autotroph = "self-nourishing"; fix inorganic carbon (eg. CO~2~) into biomass**  
**Heterotroph = consumes and metabolizes inorganic carbon (assimilates inorganic carbon)**  
**Lithotroph = breaks down inorganic substrates**  

* Based on information provided in the text and your knowledge of geography what is the deepest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this depth?  
**Deepest habitat: 4km subsurface sediments (terrestrial, probably marine as well)**  
**Limiting factor: temperaure (125 degrees Celcius; about 22 degrees/km)**  
**Mariana's Trench is 10.9 km, so deepest habitat is probably ~15km**  

* Based on information provided in the text your knowledge of geography what is the highest habitat capable of supporting prokaryotic life? What is the primary limiting factor at this height? 
**Mount Everest is 8.8km**  
**There are spore-forming bacteria 57-77km in the air**  
**Limiting factor: resources, ionizing UV radiation, lack of moisture, bad weather conditions**  
**- so there are probably no microbes actually metabolizing up there, just transporting**  
**We can estimate 20 km to be conservative, so highest is 29 km**  

* Based on estimates of prokaryotic habitat limitation, what is the vertical distance of the Earth's biosphere measured in km?  
**24 km (Mount Everest to 4 km below Mariana's Trench)**

* How was annual cellular production of prokaryotes described in Table 7 column four determined? (Provide an example of the calculation)  
**The turnover time in days is converted to years. Then, the population size is divided by the turnover rate in years.**  
*(16 days)/(365 days/year) = 0.0438 years*  
*(3.6 x 10^28^ cells)/(0.0438 years) = 8.2 x 20^29^ cells/year*

* What is the relationship between carbon content, carbon assimilation efficiency and turnover rates in the upper 200m of the ocean? Why does this vary with depth in the ocean and between terrestrial and marine habitats?  
**Carbon efficiency/growth efficiency is 20% --> they used a 4 multiplier instead of 5 (how well an organism can assimilate carbon into biomass)**  
*20fg of carbon in a prokaryotic cell --> 20^-30^ Pg/cell*  
*3.6 x 10^28^ cells x 20^-30^ Pg/cell = 0.72 Pg of C in marine heterotrophs*  
*4 x 0.72 Pg C = 2.88 Pg C/year*  
*51 Pg C/year x 0.85 (85% consumed) = 43 Pg C/year*  
*(43 Pg C/year)/(2.88 Pg C/year) = 14.9/year turnovers* OR ***1 turnover every 24.5 days* **    
**Contrasting the amount of carbon that is produced and consumed in different parts of the biosphere**  

* How were the frequency numbers for four simultaneous mutations in shared genes determined for marine heterotrophs and marine autotrophs given an average mutation rate of 4 x 10^-7^ per DNA replication? (Provide an example of the calculation with units. Hint: cell and generation cancel out)  
**The mutation rate was calulcated to the power of 4 (for 4 simultaneous mutations. Then, the total number of cells was multipled by the turnover rate to calculate cells/year. That value was multiplied by the simultaneous mutation rate to give us the number of mutations per year.**  
*4 x 10^-7^ mutations/generation*   
*(4 x 10^-7^)^4^ = 2.56 x 10^-26^ mutations/generation*  
*365/16 turnovers = 22.8 turnovers/year*  
*3.6 x 10^28^ cells x 22.8 turnovers/year = 8.2 x 10^29^ cells/year*  
*(8.2 x 10^29^ cells/year) x (2.56 x 10^-26^ mutations/generation) = 2.1 x 10^4^ mutations/year --> 0.4 hours/mutation*  

* Given the large population size and high mutation rate of prokaryotic cells, what are the implications with respect to genetic diversity and adaptive potential? Are point mutations the only way in which microbial genomes diversify and adapt?  
**Since prokaryotes have high mutation rates in such a large population, they are bound to get numerous mutations that end up persisting in the population. This leads to genetic diversity as different mutations accumulate, resulting in different strains and species. Genomes can also diversify and adapt with horizontal gene transfer.**  

* What relationships can be inferred between prokaryotic abundance, diversity, and metabolic potential based on the information provided in the text?   
**The relationship between these three can be seen as cyclic. Prokaryotes are so abundant because of their rapid growth rate and ability to survive in harsh conditions due to the amount of metabolic processes they can carry out. Because they are so abundant with a rapid growth rate, this leads to genetic diversity, which results in prokaryotes that can perform many different metabolic reactions. The fact that they have so much metabolic potential allows their abundance to continue.**  

### Problem set 02   
[Falkowsi *et al* 2008](https://www.ncbi.nlm.nih.gov/pubmed/18497287)    

#### Learning objectives  
Discuss the role of microbial diversity and formation of coupled metabolism in driving global biogeochemical cycles.

#### Specific Questions  
* What are the primary geophysical and biogeochemical processes that create and sustain conditions for life on Earth? How do abiotic versus biotic processes vary with respect to matter and energy transformation and how are they interconnected?  
**- geophysical: tectonics + atmospheric photochemical processes**  
**- biogeochemical: nitrogen cycle**  
**- abiotic: acid-base reactions (transfer of protons without electrons)**  
**- biotic: redox reactions (transfer of electrons and protons)**  
**Abiotic reactions pave the way of redox reactions**  

* Why is Earth's redox state considered an emergent property?  
**- feedback between he evolution of microbial metabolic and geochemical processes create the average redox condition of the oceans and atmosphere. It is the result of a combination of processes**  
  

* How do reversible electron transfer reactions give rise to element and nutrient cycles at different ecological scales? What strategies do microbes use to overcome thermodynamic barriers to reversible electron flow?  
**- reduction of chemicals (eg. Nitrogen) fixes them in the environment - used by higher-level organisms**   
**- synergistic cooperation of microbes; different microbes have different enzymes that have thermodynamically favourable reactions in different conditions causing a cycle**  

* Using information provided in the text, describe how the nitrogen cycle partitions between different redox "niches" and microbial groups. Is there a relationship between the nitrogen cycle and climate change?  
**3 niches**  
**- in presence of O2, NH4+ can be oxidized to nitrate in a 2-stage pathway (specific group of bacteria/archaea oxidizing ammonia to NO2-, which is then oxidized to nitrate by different nitrifying bactera)**  
**- all nitrifiers use the small differences in redox potential in the oxidation reactions to reduce CO2 to organic matter**  
**- in the absense of O2, set of microbes use NO2- and NO3- as electron acceptors in the anaerobic oxidation of organic matter, which forms N2 to complete the nitrogen cycle**  
**Climate change affects the atmospheric composition, which in turn affects which set of microorganisms (which niche) will be active. This could throw off the nitrogen cycle and make parts of it more active than we are accustomed to.**  

* What is the relationship between microbial diversity and metabolic diversity and how does this relate to the discovery of new protein families from microbial community genomes?  
**Microbial diversity and metabolic diversity are positively related. There could be limitless evolutionary diversity in nature. As new protein families are discovered, it reinforces the concept that there is a large number of undiscovered species with undiscovered protein families**  

* On what basis do the authors consider microbes the guardians of metabolism?  
**The core global gene set is dispered among microbes by either vertical or horizontal gene transfer. Boutique genes have evolved due to environmental selection, and this protects the metabolic pathways these genes encode. If it goes extinct in one unit, there will be another microbial unit in which it will likely survive.**  



### Module 01 references
Achenback J. 2012. [Spaceship Earth: A new view of environmentalism.](https://www.washingtonpost.com/national/health-science/spaceship-earth-a-new-view-of-environmentalism/2011/12/29/gIQAZhH6WP_story.html?utm_term=.4d3aeac98db8) The Washington Post.  

Canfield DE, Glazer AN, and Falkowski PG. 2010. [The Evolution and Future of Earth's Nitrogen Cycle.](https://www.ncbi.nlm.nih.gov/pubmed/20929768) Science. 330(6001):192-196.  

Falkowski PG, Fenchel T, and Delong EF. 2008. [The Microbial Engines That Drive Earth's Biogeochemical Cycles.](https://www.ncbi.nlm.nih.gov/pubmed/18497287) Science. 320(5879):1034-1039.  

Kallmeyer J *et al.*. 2012. [Global distribution of microbial abundance and biomass in subseafloor sediment.](https://www.ncbi.nlm.nih.gov/pubmed/22927371) Proc Natl Acad Sci USA. 109(40):16213-16216.   

Kasting JF and Siefert JL. 2002. [Life and the Evolution of Earth's Atmosphere.](http://science.sciencemag.org/content/296/5570/1066.long) Science. 295(5570):1066-1068.  

Leopold A. 1949. [The Land Ethic.](http://rintintin.colorado.edu/~vancecd/phil3140/Leopold.pdf) Location: Publisher. *A Sand County Almanac*. 

Mooney C. 2016. [Scientists say humans have now brought on an entirely new geologic epoch.](https://www.washingtonpost.com/news/energy-environment/wp/2016/01/07/scientists-say-humans-have-now-brought-on-an-entirely-new-geologic-epoch/?utm_term=.fe0893313ea1) The Washington Post.  

Nisbet EG and Sleep NH. 2001. [The habitat and nature of early life.](https://www.nature.com/articles/35059210) Nature. 409:1083-1091.  

Rockstrom J *et al.* 2009. [A safe operating space for humanity.](https://www.nature.com/articles/461472a) Nature. 461:472-475.  

Waters CN *et al.* 2016. [The Anthropocene is functionally and stratigraphically distinct from the Holocene](http://science.sciencemag.org/content/351/6269/aad2622) Science. 351(6269):aad2622.  

Whitman WB, Coleman DC, and Wiebe WJ. 1998. Prokaryotes: The unseen majority. Proc Natl Acad Sci USA. 95(12):6578-6583.  [PMC33863](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33863/)  



# Module 2 
## Metagenomics: Genomic Analysis of Microbial Communities  
### Evidence worksheet 04 
[Martinez *et al.* 2007](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838496/)  

#### Learning objectives  
Discuss	the	relationship	between	microbial	community	structure	and	metabolic	diversity. Evaluate	common	methods	for	studying	the	diversity	of	microbial	communities. Recognize basic	design	elements	in	metagenomic	workflows.  

#### Specific Questions:
* What were the main questions being asked?  
**How can we decribe PR photosystem genetics and biochemistry?**  
**Further characterize PR photosystem structure and function**  


* What were the primary methodological approaches used?  
**Functionally surveyed marine picoplankton large-insert genomic library for recombinant clones expressing PR photosystems in vivo**  
**Exploited transient increases in vector copy number that significantly enhanced the sensitivity of phenotypic detection**  

* Summarize the main results or findings.   
**1) Two genetically distinct recombinants (identified by their orange pigmentation) expressed a cluster of genes encoding a complete PR-based photosystem**  
**2) Heterologous expression of 6 genes (5 for photopigment biosynthetic proteins and 1 encoding a PR) generated a fully functional PR photosystem that enabled photophosphorylation in recombinant *E. coli* cells exposed to light**  
**3) A single genetic event can result in the acquisition of phototrophic capabilities in an otherwise chemoorganotrophic microorganism**  
**4) Explains the ubiquity of PR photosystems among diverse microbial taxa**  
**5) Took dna from environment, programmed heterologous host, looked for particular phenotype, discovered minimal set of genes required to express that phenotype**  

* Do new questions arise from the results?  
**How can downstream genetic and biochemical characterization using bioinformatics occur? What pathway does the PR photosystem use to utilize light to biochemical energy conversion?**  

* Were there any specific challenges or	advantages in	understanding the	 paper (e.g.	 did	 the	authors	 provide	 sufficient background information to understand experimental logic, were methods	explained	adequately,	were	any	specific	assumptions	made,	were	conclusions	justified	based	on	the	evidence,	were	the	figures	or tables useful and	easy to understand)?  
**Unsure of the purpose of inserting either one or multiple copies of the fosmid**  

### Problem set 03   
#### Learning objectives: 
Specific emphasis should be placed on the process used to find the answer. Be as comprehensive as possible e.g. provide URLs for web sources, literature citations, etc.  


#### Specific Questions:
* How many prokaryotic divisions have been described and how many have no cultured representatives (microbial dark matter)?  
**2016: 89 bacterial phyla, 20 archael phyla (via 16S rRNA databases). But could be up to 1500 bacterial phyla, as there are microbes that libe in the "shadow biosphere"**  
**2003: 26 of 52 major bacterial phyla have been cultivated**  

* How many metagenome sequencing projects are currently available in the public domain and what types of environments are they sourced from?  
**According to EBI Metagenomics, there are 110,217 data sets which consists of 10,653 metagenomes. They were sourced from multiple environments, including soil, humans, marine, freshwater, plants, and grassland.**  

* What types of on-line resources are available for warehousing and/or analyzing environmental sequence information (provide names, URLS and applications)?  
*Shotgun Metagenomics*  
**Assembly - Euler**  
**Binning - S-GCOM**  
**Annotation - KEGG**  
**Analysis pipelines - Megan 5**  
*Marker Gene Metagenomics*  
**Standalone software - OTUbase**  
**Analysis pipelines - SILVA (gold standard)**  
**Denoising - Amplicon Noise**  
**Databases - RDP (also good)**  
*Data Warehousing - Databases*  
**IMG/M, MG-RAST, NCBI/EBI which will give you sequences to feed into the aforementioned softwares**  

* What is the difference between phylogenetic and functional gene anchors and how can they be used in metagenome analysis?  
**Phylogenetic: vertigal gene transfer, carry phylogenetic information allowing tree reconstruction, taxonomix, ideally single-copy**  
**Functional: more horizontal gene transfer, identify specific biogeochemical functions associated with measurable effects, not as useful for phylogeny**  

* What is metagenomic sequence binning? What types of algorithmic approaches are used to produce sequence bins? What are some risks and opportunities associated with using sequence bins for metabolic reconstruction of uncultivated microorganisms?  
**The process of grouping of sequences that come from a single genome**  
**Types of algorithms: align sequences to database, group to each other based in DNA characteristics (GC content, codon usage)**  
**Risks: incomplete coverage of genome sequence, contamination from different phylogeny**  

* Is there an alternative to metagenomic shotgun sequencing that can be used to access the metabolic potential of uncultivated microorganisms? What are some risks and opportunities associated with this alternative?  
**Functional screens (biochmical assays, etc.), 3rd generation sequencing (nanopore), single cell sequencing, FISH probe**  

### Module 02 references
Madsen EL. 2005. [Identifying microorganisms responsible for ecologically significant biogeochemical processes](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838496/) Nature Reviews Microbiology. 104(13):5590-5595.

Martinwz A, Bradley AS, Waldbauer JR, Summons RE, and DeLong EF. 2007. [Proteorhodopsin photosystem gene expression enables photophosphorylation in a heterologous host](https://www.nature.com/articles/nrmicro1151) Proc Natl Acad Sci USA. 3:439-446.

Wooley JC, Godzik A, and Friedberg I. 2010. [A Primer on Metagenomics](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000667) PLoS Computational Biology. 6(2).

# Module 3  
## Microbial Species Concepts  
### Evidence worksheet 05
[Welch RA *et al.* 2002](https://www.ncbi.nlm.nih.gov/pubmed/12471157)  

#### Learning objectives part 1: 
Evaluate the concept of microbial species based on environmental surveys and
cultivation studies. Explain the relationship between microdiversity, genomic diversity and metabolic
potential. Comment on the forces mediating divergence and cohesion in natural microbial
communities


#### Specific Questions:
* What were the main questions being asked?  
**The different genes involved in E. coli pathogenicity and the difference between different strains of E. coli**  
 

* What were the primary methodological approaches used?  
**Isolate from a hospital patient**  
**Random clones sequenced using dye-terminator chemistry and data collected on ABI sequencers**  
**Sequence data assembled by SEQMANII**  
**Whole genome XhoI optical map for ordering of contigs and providing physical map of whole genome**  
**Annotations using MAGPIE**  


* Summarize the main results or findings. 
**Only 39% of combined sets of proteins between a uropathic, enterohemorrhagic, and laboratory strain E. coli are common to all 3 strains**  


* Do new questions arise from the results?  


* Were there any specific challenges or	advantages in	understanding the	 paper (e.g.	 did	 the	authors	 provide	 sufficient background information to understand experimental logic, were methods	explained	adequately,	were	any	specific	assumptions	made,	were	conclusions	justified	based	on	the	evidence,	were	the	figures	or tables useful and	easy to understand)?	

#### Learning objectives part 2:  
Comment	on	 the	creative	 tension	between	gene	loss,	duplication	and	acquisition	as	it	
relates	to	microbial	genome	evolution. Identify	common	molecular	signatures	used	to	infer	genomic	identity	and	cohesion. Differentiate	between	mobile	elements	and	different	modes	of	gene	transfer.  

#### Specific Questions:  
Based	on	your	reading	and	discussion	notes,	explain	the	meaning	and content	of	the	following	figure	derived	from	the	comparative	genomic	analysis	of	three E.	coli genomes	by	Welch	et	al.	
Remember	 that	 CFT073	 is	 a	 uropathogenic	 strain	 and that	 EDL933	 is	 an	 enterohemorrhagic	strain.	 Explain	 how	 this	 study	 relates	 to	 your understanding	 of	 ecotype	 diversity. Provide	 a	definition	of	ecotype	in	the	context	of	the human	body.	Explain	why	certain	subsets	of	genes	in	CFT073	 provide	 adaptive	 traits under	 your	 ecological	 model and	 speculate	 on	 their	 mode	 of	vertical	descent	or	gene transfer.

### Problem set 04
See .Rmd and .html files "Problem_set_04_19163147" in portfolio repo

### Module 03 references
Welch RA *et al.* 2002. [Extensive mosaic structure revealed by the complete genome sequence of uropathogenic Escherichia coli](https://www.ncbi.nlm.nih.gov/pubmed/12471157) Proc Natl Acad Sci USA. 99(26):17020-17024.
